libglnx

```diff
@@ -1 +1 @@
-Subproject commit a6d08657aa868a0d5c7b5dd494e16f65415a148f
+Subproject commit afe3c3a86178c29ceaa3a5e46397ab2fa97202b3
```

Comment:
Fixed in 3315b86.

Not sure why this happened in the first place... I guess git's submodule handling still sucks.